### PR TITLE
Fix poetry not finding the pwndbg folder

### PR DESCRIPTION
**Pull request by k4lizen**
Manifested with error message:
```
Detected outdated Pwndbg dependencies (poetry.lock). Updating. 
Poetry could not find a pyproject.toml file in <calling dir> or its parents
```
when pwndbg gets sourced from gdb.

### CODE REVIEW

**Comment by patryk4815** [comment]
@k4lizen can you fix also lldbinit ;)?

File: gdbinit.py [hunk 1]
```diff
@@ -29,12 +29,14 @@ def hash_file(file_path: str | Path) -> str:
     return file_hash.hexdigest()
 
 
-def run_poetry_install(poetry_path: os.PathLike[str], dev: bool = False) -> Tuple[str, str, int]:
+def run_poetry_install(
+    poetry_path: os.PathLike[str], src_root, dev: bool = False
```
Comment (patryk4815) [inline gdbinit.py]: `src_root` type is missing 😫

**Comment by k4lizen** [comment]
done

**Review by patryk4815 — APPROVED** [review]
👍

**Comment by disconnect3d** [comment]
Awesome, thanks! I wondered why sometimes poetry gave weird warnings/errors when running Pwndbg 😓

**Pull request merged**